data-model/build.gradle

```diff
@@ -0,0 +1,3 @@
+description 'Shared POJOs and interfaces for the Blox DataService'
```

Comment:
Do you have a suggestion? What do we call the packages that contain the DAO POJOs and Java interfaces that define each Lambda microservice?